### DISCUSSION TITLE
Lightning Node Circular Rebalancing Fee Rate:

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [lightningnetwork/lnd#7979] [Q&A]
Hello All,

I have a long quick question please.

Can anyone here please tell me how I can find out when the Miner's Fees are at their lowest so I can minimize the amount of sats we need to pay when performing a circular re balance. Currently it only seems to work when we choose a percentage like 1 or 2 % with the amount of sats required for it to actually work set at a very low amount, we initially tried the fee rate type of re balance but that does not seem to ever work? At the moment we are paying more in fees to re balance than we are able to bring in for routing. We hope to get our node's financial situation in a more balanced position but routing does not seems to flow really well unless our node's fee rate is really low?.

Thank you. :)

Orange_Bonzo

**Comment by bitromortac** [comment]
Circular rebalances are purely off-chain, so no mining fees are involved (only routing fees). It is often the case that channels get unbalanced into the direction of the general network flow. If you try to reverse that it usually costs a high fee. Try to increase the fee rate for the outgoing channels, but this is not a straight-forward process (you can try out autofees on https://terminal.lightning.engineering/ and/or augment rebalancing with looping (https://docs.lightning.engineering/lightning-network-tools/loop)).

**Comment by BonzoDogDooDah** [comment]
Hi,

Thank you for the information I will endeavour to put it to good use.

Orange_Bonzo
-- 
 Sent with Tutanota; enjoy secure & ad-free emails: 
 https://tutanota.com


18 Sept 2023, 21:27 by ***@***.***:
